### ISSUE TITLE
Resets to top of scrollview when moving from one TextInput to another

### ISSUE THREAD

**Issue by joelgetaction** [APSL/react-native-keyboard-aware-scroll-view#329]
It seems like issue https://github.com/APSL/react-native-keyboard-aware-scroll-view/issues/82 is back.  I am moving between TextInputs and when I do, KeyboardAwareScrollView jumps up to the top of the scroll view and then scrolls to the next input.  Or sometimes scrolls to the next input and then jumps up to the top and then down.  Setting `enableResetScrollToCoords={false}` doesn't help.

Latest version of RN and this module.

Any ideas how to fix?  Thanks for this awesome component by the way, very useful!  :-)

**Comment by joelgetaction** [comment]
Hmm, it seems to behave a lot better if I set `blurOnSubmit={false}` on my `TextInput` elements, but then they don't behave as desired if the user just taps on a single input as opposed to next-buttoning between them ... Just wanted to add this in case it might help others.

**Comment by joelgetaction** [comment]
Update: `blurOnSubmit` should be `false` for iOS and `true` for Android.  Setting it to false on Android totally breaks the scrolling.

**Comment by alvaromb** [comment]
Please try this on a real device, the jump to top issue is affects only to the simulator.

**Comment by alvaromb** [comment]
I'll try to reproduce this today. Thanks for reporting.

**Comment by alvaromb** [comment]
I'm unable to reproduce the issue on a real device in an app we're about to release. Can you share the source code of your component?

**Comment by joelgetaction** [comment]
I can’t share the source code of the actual app because it’s a private app but I could rip out the relevant components and put them in a small test app that reproduces the issue. Give me a few days to do that please and I’ll update this issue with a link to that repo when it’s done ...
  

  
  

  
  
>

**Comment by wbercx** [comment]
> Please try this on a real device, the jump to top issue is affects only to the simulator.

From the way you worded that, it sounds like it's a known issue. 

Do you know why it happens in the Simulator and how to avoid it? I've got a pretty long form in my app and am getting pretty close to defenestrating my Mac :smile:

**Comment by joelgetaction** [comment]
@wbercx and @alvaromb - I definitely see this issue on a real device.  It does not happen only on the simulator ...

**Comment by wbercx** [comment]
> @wbercx and @alvaromb - I definitely see this issue on a real device. It does not happen only on the simulator ...

Yeah, just tested it on an iPad. Happens there too. `enableResetScrollToCoords={false}` on my `KeyboardAwareFlatList` has actually worked for me somewhat.

**Comment by szilarddoro** [comment]
I could fix this issue by adding `keyboardOpeningTime={-1}` to `KeyboardAwareScrollView`. Seems a little hacky but solved the problem in my case.

**Comment by krisxcrash** [comment]
seems this issue is related to https://github.com/APSL/react-native-keyboard-aware-scroll-view/issues/333 - has there been any fix implemented for this? 

`keyboardOpeningTime={-1}` did not work
`enableResetScrollToCoords={false}` did not work

` enableAutomaticScroll `was the closest I could get to a fix but it still scrolls down slightly when the input is focused.

**Comment by TalonHughes13** [comment]
This happens to me as well with the following properties set:
`keyboardOpeningTime={0}`
`enableResetScrollToCoords={false}`
`keyboardShouldPersistTaps="handled"`
`extraScrollHeight={120}`

Pressing the second input a second time, appears to work, but pressing it the first time it does bounce or scroll strangely.

**Comment by asoomar** [comment]
I am also running into the same exact issue. I tried all the things above with no luck. If one textinput is selected and I select the next one, it adjusts as expected, but then all of the sudden it just scrolls all the way up. Has anyone else figured out a way to fix this? This is happening on a simulated device and real device.

**Comment by Return-1** [comment]
Im also experiencing this. Have no clue how to approach it

**Comment by caglardurmus** [comment]
> `keyboardOpeningTime={0}`
> `enableResetScrollToCoords={false}`

worked for me

**Comment by Technaur** [comment]
Has anyone found a solution for this yet? Using RN 0.63.3 and react-native-keyboard-aware-scroll-view 0.9.4

**Comment by richlewis14** [comment]
> This happens to me as well with the following properties set: `keyboardOpeningTime={0}` `enableResetScrollToCoords={false}` `keyboardShouldPersistTaps="handled"` `extraScrollHeight={120}`
> 
> Pressing the second input a second time, appears to work, but pressing it the first time it does bounce or scroll strangely.

Same issue here for me, did you manage to fix this at all?

**Comment by achall9** [comment]
Getting this issue as well on real device.

**Comment by L4liK0rn** [comment]
Same issue happening for me, even on multiple real iOS devices.
It seems current best option is the setTimeout.

**Comment by DaltonPelkey** [comment]
Just updating here. I had this issue a few weeks ago, and although I can't remember where I found the work-around, this is what works for me:
`keyboardOpeningTime={Number.MAX_SAFE_INTEGER}`

**Comment by BuiHung1612** [comment]
> Just updating here. I had this issue a few weeks ago, and although I can't remember where I found the work-around, this is what works for me: `keyboardOpeningTime={Number.MAX_SAFE_INTEGER}`

it worked without scrolling down. Thank you very much

**Comment by danielfogacaa** [comment]
> Just updating here. I had this issue a few weeks ago, and although I can't remember where I found the work-around, this is what works for me: `keyboardOpeningTime={Number.MAX_SAFE_INTEGER}`

Worked for me too. Thank you

**Comment by hakudevtw** [comment]
> > Just updating here. I had this issue a few weeks ago, and although I can't remember where I found the work-around, this is what works for me: `keyboardOpeningTime={Number.MAX_SAFE_INTEGER}`
> 
> it worked without scrolling down. Thank you very much

You saved my day, thanks a lot ! This works !

**Comment by bmitioglov** [comment]
> Just updating here. I had this issue a few weeks ago, and although I can't remember where I found the work-around, this is what works for me: `keyboardOpeningTime={Number.MAX_SAFE_INTEGER}`

You are the BEST man!!! Thank you sooooo much!

**Comment by saintyoseph** [comment]
> Just updating here. I had this issue a few weeks ago, and although I can't remember where I found the work-around, this is what works for me: `keyboardOpeningTime={Number.MAX_SAFE_INTEGER}`

I was trying to solve this problem for like 3+ hours and finally found this. Thank you!

**Comment by daregula** [comment]
> Just updating here. I had this issue a few weeks ago, and although I can't remember where I found the work-around, this is what works for me: `keyboardOpeningTime={Number.MAX_SAFE_INTEGER}`

worked thanks so much mate
